### DISCUSSION TITLE
Gerar apresentação de slides customizadas

### DISCUSSION THREAD

**Discussion by hugojardim** [Flutterando/forum#198] [Perguntas e Respostas]
Boa tarde pessoal, estou com dificuldades de criar uma apresentação de slides customizada com flutter. O único package que consegui ver alguma utilidade nesse sentido foi o flutter_pptx que é equivalente ao dart_pptx.  Porém, nos meus testes ele tem uma customização bem limitada, alcançando no máximo a adição em cada slide o título, subtítulo, texto, lista, imagens e imagem de background, não deixando customizar a posição de cada item nem customizar as fontes.

**Comment by eduardoflorence** [comment]
Oi Hugo, eu gosto muito desse package: https://github.com/leoafarias/superdeck
